{"name": "test_login_button[chromium]", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible", "trace": "self = <Suites.Login_Signup.login.LoginButton object at 0x0000023D37098F50>\n\n    @allure.step(\"Enter account\")\n    def enter_account(self):\n        try:\n            account_button = self.page.get_by_role(\"button\", name=\"Account\")\n>           account_button.click()\n\nSuites\\Login_Signup\\login.py:194: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:14851: in click\n    self._sync(\n.venv\\Lib\\site-packages\\playwright\\_impl\\_locator.py:153: in click\n    return await self._frame.click(self._selector, strict=True, **params)\n.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:488: in click\n    await self._channel.send(\"click\", locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x0000023D3575EE70>, cb = <function Channel.send.<locals>.<lambda> at 0x0000023D3716B600>, is_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nE           Call log:\nE           waiting for get_by_role(\"button\", name=\"Account\")\n\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:513: TimeoutError\n\nThe above exception was the direct cause of the following exception:\n\nself = <Suites.Login_Signup.login.LoginButton object at 0x0000023D37098F50>\n\n    @allure.step(\"Enter account\")\n    def enter_account(self):\n        try:\n            account_button = self.page.get_by_role(\"button\", name=\"Account\")\n            account_button.click()\n            allure.attach(self.page.screenshot(), name='Account button pressed',\n                          attachment_type=allure.attachment_type.PNG)\n    \n        except Exception as e:\n            allure.attach(self.page.screenshot(), name='Account button is not pressed',\n                          attachment_type=allure.attachment_type.PNG)\n>           raise AssertionError from e\nE           AssertionError\n\nSuites\\Login_Signup\\login.py:201: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\npage = <Page url='https://tombriches.com/'>\n\n    @allure.suite(\"Login_Signup via login button\")\n    def test_login_button(page):\n      login = LoginButton(page)\n>     login.open_site()\n\ntest_tr_smoke.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nSuites\\Login_Signup\\login.py:130: in open_site\n    self.press_log_in_button()\nSuites\\Login_Signup\\login.py:144: in press_log_in_button\n    self.fill_email()\nSuites\\Login_Signup\\login.py:158: in fill_email\n    self.fill_password_field()\nSuites\\Login_Signup\\login.py:172: in fill_password_field\n    self.click_log_in_button()\nSuites\\Login_Signup\\login.py:188: in click_log_in_button\n    self.enter_account()\nSuites\\Login_Signup\\login.py:204: in enter_account\n    self.check_email()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Suites.Login_Signup.login.LoginButton object at 0x0000023D37098F50>\n\n    @allure.step(\"Check email\")\n    def check_email(self):\n        email = self.page.get_by_text(\"samoilenkofluttershy@gmail.com\")\n    \n        if email.is_visible():\n            pass\n        else:\n>           raise AssertionError(\"Email is not visible\")\nE           AssertionError: Email is not visible\n\nSuites\\Login_Signup\\login.py:213: AssertionError"}, "steps": [{"name": "Press log in button", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 144, in press_log_in_button\n    self.fill_email()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 158, in fill_email\n    self.fill_password_field()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 172, in fill_password_field\n    self.click_log_in_button()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 188, in click_log_in_button\n    self.enter_account()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 204, in enter_account\n    self.check_email()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 213, in check_email\n    raise AssertionError(\"Email is not visible\")\n"}, "steps": [{"name": "Fill in credentials", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 158, in fill_email\n    self.fill_password_field()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 172, in fill_password_field\n    self.click_log_in_button()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 188, in click_log_in_button\n    self.enter_account()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 204, in enter_account\n    self.check_email()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 213, in check_email\n    raise AssertionError(\"Email is not visible\")\n"}, "steps": [{"name": "Fill in password", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 172, in fill_password_field\n    self.click_log_in_button()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 188, in click_log_in_button\n    self.enter_account()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 204, in enter_account\n    self.check_email()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 213, in check_email\n    raise AssertionError(\"Email is not visible\")\n"}, "steps": [{"name": "Click log in button", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 188, in click_log_in_button\n    self.enter_account()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 204, in enter_account\n    self.check_email()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 213, in check_email\n    raise AssertionError(\"Email is not visible\")\n"}, "steps": [{"name": "Enter account", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 204, in enter_account\n    self.check_email()\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 213, in check_email\n    raise AssertionError(\"Email is not visible\")\n"}, "steps": [{"name": "Check email", "status": "failed", "statusDetails": {"message": "AssertionError: Email is not visible\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Login_Signup\\login.py\", line 213, in check_email\n    raise AssertionError(\"Email is not visible\")\n"}, "start": 1714566164112, "stop": 1714566164124}], "attachments": [{"name": "Account button is not pressed", "source": "acdf9007-b822-4b55-ab81-6af5754340cf-attachment.png", "type": "image/png"}], "start": 1714566128920, "stop": 1714566164125}], "attachments": [{"name": "Log in button is not clicked", "source": "c72e7dbf-f1af-4da4-8265-6f23d93194f0-attachment.png", "type": "image/png"}], "start": 1714566093336, "stop": 1714566164125}], "attachments": [{"name": "Password field filled", "source": "125bfead-37b2-4837-bd1c-7349b2bfcf42-attachment.png", "type": "image/png"}], "start": 1714566093051, "stop": 1714566164126}], "attachments": [{"name": "Username field filled", "source": "31668b98-07b1-4530-a16d-13a81598758e-attachment.png", "type": "image/png"}], "start": 1714566092874, "stop": 1714566164127}], "attachments": [{"name": "Log in button clicked", "source": "2e5c459b-c50d-4162-9356-55e604c325ac-attachment.png", "type": "image/png"}], "start": 1714566092541, "stop": 1714566164127}], "attachments": [{"name": "Page open", "source": "06b710b0-d6cc-4ccc-a315-ba34fa8d4737-attachment.png", "type": "image/png"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1714566089981, "stop": 1714566164128, "uuid": "a8fcefb3-18e4-4cae-ab69-d368853e4aba", "historyId": "7c10a9fdbdc557b6f1a1ea648e01553e", "testCaseId": "69e36fe8a4f17c9ffcf2589e4624efa8", "fullName": "test_tr_smoke#test_login_button", "labels": [{"name": "suite", "value": "Login_Signup via login button"}, {"name": "host", "value": "DESKTOP-EFHMBBJ"}, {"name": "thread", "value": "30928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_tr_smoke"}]}